src/main/kotlin/com/noumenadigital/npl/cli/CommandProcessor.kt

```diff
@@ -1,34 +1,31 @@
 package com.noumenadigital.npl.cli
 
+import com.noumenadigital.npl.cli.exception.CommandExecutionException
 import com.noumenadigital.npl.cli.exception.CommandNotFoundException
 import com.noumenadigital.npl.cli.exception.CommandParsingException
 import com.noumenadigital.npl.cli.exception.InternalException
 import com.noumenadigital.npl.cli.exception.buildOutputMessage
 import com.noumenadigital.npl.cli.service.CommandsParser
 import java.io.Writer
+import java.nio.file.Path
 
 class CommandProcessor(
     private val commandsParser: CommandsParser = CommandsParser,
+    private val baseDir: Path? = null,
 ) {
-    companion object {
-        private const val START_MESSAGE = "Executing command '%s'...\n"
-        private const val END_MESSAGE_SUCCESS = "\nCommand '%s' finished SUCCESSFULLY."
-    }
```

Comment:
Removed these because I don't think other CLI tools typically print like this. Especially the last line is confusing if you have e.g. warnings and a summary of that above it.